package.json

```diff
@@ -21,8 +21,10 @@
   ],
   "scripts": {
     "clean": "rimraf __tests__/write-test* coverage",
-    "jest": "jest --coverage",
-    "test": "run-s clean jest"
+    "jest": "jest --coverage ",
+    "jest-verbose": "jest --coverage --verbose",
+    "test": "run-s clean jest",
+    "test-verbose": "run-s clean jest-verbose"
```

Comment:
Any chance I could ask you to use colons instead of hyphens for the script names? I don't have much of a reason other than personal preference 😃.

For example, `test-verbose` becomes `test:verbose`, and so on.

Reply:
Done, with a rebase